src/twisted/words/protocols/irc.py

```diff
@@ -1197,6 +1197,16 @@ class IRCClient(basic.LineReceiver):
     @ivar heartbeatInterval: Interval, in seconds, to send I{PING} messages to
         the server as a form of keepalive, defaults to 120 seconds. Use L{None}
         to disable the heartbeat.
+
+    @type decodeCodecs: C{list} of C{str}
```

Comment:
FYI for the future, you can omit the `@type` when there's a type annotation on the variable itself &mdash; Pydoctor will use the annotation now!